roles/caliopen/templates/configs/apiv2.yaml

```diff
@@ -5,41 +5,49 @@ ApiConfig:
+      - {{ hostvars[host]['backend_ip'] }}
+{% endfor %}
       keyspace: caliopen
       consistency_level: 1
       raw_size_limit: 1048576                                # max size in bytes for objects in db. Use S3 interface if larger.
       object_store: s3
       object_store_settings:
-        endpoint: minio:9090
-        access_key: CALIOPEN_ACCESS_KEY_                     # Access key of 5 to 20 characters in length
-        secret_key: CALIOPEN_SECRET_KEY_BE_GOOD_AND_LIVE_OLD # Secret key of 8 to 40 characters in length
-        location: eu-fr-localhost                            # S3 region.
+        endpoint: {{ hostvars['object_store1']['backend_ip'] }}:9000
+        access_key: {{ object_store_access_key }}
+        secret_key: {{ object_store_secret_key }}
+        location: eu-fr-paris
         buckets:
           raw_messages: caliopen-raw-messages                # bucket name to put raw messages to
           temporary_attachments: caliopen-tmp-attachments    # bucket name to store draft attachments
+      use_vault: true
+      vault_settings:
+        url: https://vault.{{ caliopen_domain_name }}:8200
+        username: {{ vault_api_username }}
+        password: {{ vault_api_password }}
   IndexConfig:
     index_name: elasticsearch
     index_settings:
       hosts:
-      - http://elasticsearch:9200
+{% for host in groups['store'] %}
+      - http://{{ hostvars[host]['backend_ip'] }}:9200
+{% endfor %}
   NatsConfig:
-    url: nats://nats:4222
+    url: nats://{{ hostvars['mq1']['backend_ip'] }}:4222
     outSMTP_topic: outboundSMTP     # topic's name to post "send" draft order
     contacts_topic: contactAction   # topic's name to post messages regarding contacts' events
-  swaggerSpec: ./swagger.json #absolute path or relative path to go.server bin
+  swaggerSpec: /etc/caliopen/swagger.json #absolute path or relative path to go.server bin
   RedisConfig:
-    host: redis:6379
-    password: ""        #no password set
-    db: 0               #use default db
+    host: {{ hostvars['cache1']['backend_ip'] }}:6379
+    password: ""      #no password set
+    db: 0             #use default db
   NotifierConfig:
-    base_url: http://localhost:4000                         # url upon which to build custom links sent to users. NO trailing slash please.
-    admin_username: admin                                   # username on whose behalf notifiers will act. This admin user must have been created before by other means.
-    templates_path: "../defs/notifiers/templates/"          # path to yaml/j2 templates directory, WITH trailing slash please.
+    base_url: https://{{ caliopen_domain_name }}
+    admin_username: admin
+    templates_path: "/etc/caliopen/notifiers/templates/"
 ProxyConfig:
   host: 0.0.0.0
   port: 31415
   routes:
   - /api/v2/: 127.0.0.1:6544
-  - /: apiv1:6543         # route "/" catch all other requests
-
+  - /: {{ hostvars['api1']['backend_ip'] }}:6543         # route "/" catch all other requests
```

Comment:
Not sure how this one goes...

Reply:
we will not use the proxy of apiv2. We must figure out how to correctly disable it